src/api/crypto/Seed.cpp

```diff
@@ -770,13 +755,11 @@ auto Seed::SetDefault(const Identifier& id) const noexcept -> bool
     }
 
     const auto seedID = id.str();
-    const auto exists = [&] {
-        for (const auto& [value, alias] : api_.Storage().SeedList()) {
-            if (value == seedID) { return true; }
-        }
 
-        return false;
-    }();
+    auto exists = false;
+    for (const auto& [value, alias] : api_.Storage().SeedList()) {
+        if (value == seedID) { exists = true; }
```

Comment:
The logic has changed. In the original, the lambda returned with true on the first match, which was correct as we were testing the existence. Here, the value of exists will depend on the last comparison. Suggestuib: { exists = true; break; }

Reply:
The logic is the same. You described situation  exist = (value == seedID) ? true : false; In our code false is assigned only once, before the loop and condition check. Anyway break is good idea for optimization rezones and I'll add it

Reply:
True.

Reply:
done